hydromt/models/model.py

```diff
@@ -135,7 +137,11 @@ def __init__(
         self._components: Dict[str, ModelComponent] = {}
         self._add_components(components)
 
-        config_component: ConfigComponent = ConfigComponent(self, config_fn=config_fn)
+        config_component: ConfigComponent = ConfigComponent(
+            self,
+            filename=config_filename,
+            default_template_filename=default_config_template_filename,
+        )
```

Comment:
I don't know if we want to add a config component by default to Model, especially some model might decide to create a DelwaqConfigComponent just because they have to subclass as the format is not toml or yaml.

Reply:
sure I will remove it